### DISCUSSION TITLE
Proposal for getting started tasks for newbies.

### DISCUSSION THREAD

**Discussion by dataroaring** [apache/doris#8674] [RFC]
There are many codes in the Doris project,  it is difficult for newbies to get started and participate.  We should maintain a list of small and simple tasks for newbies to participate in the Doris. We can help newbies to improve engineering ability and develop ability to write good document, clean codes and unit tests. As a result, we can cultivate more developers for Doris.

Tasks collected from discussions below.
### NEW FEATURES
- [ ] nth_value function
- [ ] collect_set function

### IMPROVEMENT

- [ ] recover ddl https://github.com/apache/incubator-doris/issues/8421
- [ ] routine load with sasl_ssl https://github.com/apache/incubator-doris/discussions/7861

### TESTS 
write cases in sql under regression-test, please refer to regression-test/suite/empty_table in pr https://github.com/apache/incubator-doris/pull/8681

- [ ] HLL related
- [ ] BITMAP related
- [ ] Aggregated function
- [ ] Decimal
- [ ] Double
- [ ] Set
- [ ] ...

**Comment by jackwener** [comment]
It's a great ideas. BTW, It would be better to the `newer` instead of `student`. 

It would be nice to provide a started task for all new contributors, not just students.

**Comment by dataalive** [comment]
tasks : compare hive functions, we could give a list that Doris has not implemented. like 

- nth_value  
- collect_set

**Comment by dataroaring** [comment]
1. recover ddl . https://github.com/apache/incubator-doris/issues/8421

**Comment by wangbo** [comment]
How about start by adding uts and regression tests for existing modules or modules in development?

**Reply by dataroaring** [reply]
great!

**Comment by kpfly** [comment]
good idea

wangbo ***@***.***> 于2022年3月28日周一 13:58写道：

**Comment by morningman** [comment]
https://github.com/apache/incubator-doris/issues/7582

**Comment by dataalive** [comment]
https://github.com/apache/doris/issues/10473

**Comment by dataalive** [comment]
https://github.com/apache/doris/issues/11331
